linked_list/implementation.py

```diff
@@ -1,40 +1,90 @@
 from .interface import AbstractLinkedList
+from .node import Node
 
+# Don't think I'm as far off as the tests make it seem 
+# Believe the append is working correctly once the TypeError and eq are cleaned up
+<<<<<<< HEAD
+# Haven't started pop functionality because I know I'll need get_item
+# Can't test get_item until 'NoneType' issue is resolved
+=======
+# Haven't started pop functionality
+#
+>>>>>>> 2eb5873dc8c0c4940ff240a611eb5ec49b5c35ae
 
 class LinkedList(AbstractLinkedList):
     """
     Implementation of an AbstractLinkedList inteface.
     """
 
     def __init__(self, elements=None):
-        pass
-
+        # if elements defaulted to None start and end (needed in tests) default to None as well for error handling
+        self.start = None
+        self.end = None
+        # Now where/how to populate these? >> assuming the append and iter come into play here >> yep.
+        # Need to build list of nodes using append starting at start until >> self.start is None b/c out of elements
+        # elements constructor
+        self.elements = [self.append(elem) for elem in elements if elements]
```

Comment:
Another solution would be `[self.append(elem) for elem in elements or []]`. `elements or []` evaluates to `None or []` which finally returns an empty list.